### PR TITLE
API Start

### PR DESCRIPTION
**Pull request by darthcod3r**
Worked on API and added DBContext. Amended current data models to use long instead of int for id values.

### CODE REVIEW

**Review by ParkerW98 — DISMISSED** [review]
Looks great!

**Review by snholmes — COMMENTED** [review]
Looks great! Good work.

**Review by Bansenauer-Cascadia — APPROVED** [review]
Good work

**Review by ParkerW98 — APPROVED** [review]
good to go!

File: SuiteCareers/SuiteCareers/Program.cs [hunk 1]
```diff
@@ -5,13 +5,14 @@
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
-var connectionString = builder.Configuration.GetConnectionString("DefaultConnection") ?? throw new InvalidOperationException("Connection string 'DefaultConnection' not found.");
-builder.Services.AddDbContext<ApplicationDbContext>(options =>
+var connectionString = builder.Configuration.GetConnectionString("SuiteCareers-Mac-Sqlite") ?? throw new InvalidOperationException("Connection string 'DefaultConnection' not found.");
```
Comment (tylerjacobwilson) [inline SuiteCareers/SuiteCareers/Program.cs:8]: Are we going to be switching back and forth between SQL lite and sqlserver, is there a way around this.

**Review by snholmes — APPROVED** [review]
Great work! These changes look good. Here's my stamp of approval.

**Review by tylerjacobwilson — APPROVED** [review]

**Pull request merged**